ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkReader.java

```diff
@@ -249,12 +249,29 @@ private Void readPhaseTapChanger(String[] tokens) {
         int num = Integer.parseInt(tokens[1]);
         int tap = Integer.parseInt(tokens[2]);
         String id = mapper.getId(AmplSubset.PHASE_TAP_CHANGER, num);
-        TwoWindingsTransformer twt = network.getTwoWindingsTransformer(id);
-        if (twt == null) {
-            throw new AmplException("Invalid two windings transformer id '" + id + "'");
+        if (id.endsWith(AmplConstants.LEG1_SUFFIX) || id.endsWith(AmplConstants.LEG2_SUFFIX) || id.endsWith(AmplConstants.LEG3_SUFFIX)) {
+            ThreeWindingsTransformer twt = network.getThreeWindingsTransformer(id.substring(0, id.indexOf(AmplConstants.LEG2_SUFFIX)));
```

Comment:
I'm surprised by the end of this statement: what happen if id endsWith LEG1 or LEG3 ?

Reply:
I add a test and make a fix. Just tell me if it is okay for you.